apps/arbeidsforhold-service/src/main/java/no/nav/registre/testnorge/arbeidsforholdservice/domain/v2/ArbeidsforholdResponse.java

```diff
@@ -0,0 +1,150 @@
+package no.nav.registre.testnorge.arbeidsforholdservice.domain.v2;
+
+import lombok.AllArgsConstructor;
+import lombok.Builder;
+import lombok.Getter;
+import lombok.NoArgsConstructor;
+import lombok.Setter;
+
+import java.time.LocalDate;
+import java.time.LocalDateTime;
+import java.util.ArrayList;
+import java.util.List;
+
+import static java.util.Objects.isNull;
+
+@Getter
+@Setter
+@Builder
+@NoArgsConstructor
+@AllArgsConstructor
+public class ArbeidsforholdResponse {
+
+    private Ansettelsesperiode ansettelsesperiode;
+    private List<AntallTimerForTimeloennet> antallTimerForTimeloennet;
+    private List<Arbeidsavtale> arbeidsavtaler;
+    private String arbeidsforholdId;
+    private Arbeidsgiver arbeidsgiver;
+    private Arbeidstaker arbeidstaker;
+    private boolean innrapportertEtterAOrdningen;
+    private Long navArbeidsforholdId;
+    private Arbeidsgiver opplysningspliktig;
+    private Fartoy fartoy;
+    private List<PermisjonPermittering> permisjonPermitteringer;
+    private LocalDateTime registrert;
+    private LocalDateTime sistBekreftet;
+    private String type;
+    private List<Utenlandsopphold> utenlandsopphold;
+
+    public List<PermisjonPermittering> getPermisjonPermitteringer() {
+        if (isNull(permisjonPermitteringer)) {
+            permisjonPermitteringer = new ArrayList<>();
+        }
+        return permisjonPermitteringer;
+    }
+
+    @Getter
+    @Setter
+    @Builder
+    @NoArgsConstructor
+    @AllArgsConstructor
+    public static class Utenlandsopphold {
```

Comment:
Kan du lege dette ut til egene objecter? Blir litt for mye i en fil.

Reply:
Er det ikke ryddig å ha det samlet, gir god oversikt, eller?